src/polyswarm_api/endpoint.py

```diff
@@ -58,7 +58,8 @@ def execute(self):
             self.request_parameters.setdefault('stream', True)
         self.raw_result = self.session.request(**self.request_parameters)
         logger.debug('Request returned code %s with content:\n%s',
-                     self.raw_result.status_code, self.raw_result.content)
+                      self.raw_result.status_code, self.raw_result.content if not self.request_parameters.get('stream', False)
+                     else '')
```

Comment:
Consider to be explicit on the debug message
```suggestion
                     else '(steaming)')
```

Reply:
I would actually just remove the content, just print the return code.